internal/provider/schema_gitlab_application_settings.go

```diff
@@ -0,0 +1,3054 @@
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"pipeline_limit_per_project_user_sha": {
+			Description: "Maximum number of pipeline creation requests per minute per user and commit. Disabled by default.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"plantuml_enabled": {
+			Description: "(If enabled, requires: plantuml_url) Enable PlantUML integration. Default is false.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"plantuml_url": {
+			Description: "The PlantUML instance URL for integration.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"polling_interval_multiplier": {
+			Description: "Interval multiplier used by endpoints that perform polling. Set to 0 to disable polling.",
+			Type:        schema.TypeFloat,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"project_export_enabled": {
+			Description: "Enable project export.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"prometheus_metrics_enabled": {
+			Description: "Enable Prometheus metrics.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"protected_ci_variables": {
+			Description: "CI/CD variables are protected by default.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"push_event_activities_limit": {
+			Description: "Number of changes (branches or tags) in a single push to determine whether individual push events or bulk push events are created. Bulk push events are created if it surpasses that value.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"push_event_hooks_limit": {
+			Description: "Number of changes (branches or tags) in a single push to determine whether webhooks and services fire or not. Webhooks and services aren’t submitted if it surpasses that value.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"rate_limiting_response_text": {
+			Description: "When rate limiting is enabled via the throttle_* settings, send this plain text response when a rate limit is exceeded. ‘Retry later’ is sent if this is blank.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"raw_blob_request_limit": {
+			Description: "Max number of requests per minute for each raw path. Default: 300. To disable throttling set to 0.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"user_email_lookup_limit": {
+			Description: "caution Deprecated in GitLab 14.9 will be removed in 15.0. Replaced by search_rate_limit. Max number of requests per minute for email lookup. Default: 60. To disable throttling set to 0.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
```

Comment:
```suggestion
			Description: "Deprecated in GitLab 14.9, will be removed in 15.0. Replaced by search_rate_limit. Max number of requests per minute for email lookup. Default: 60. To disable throttling set to 0.",
			Type:        schema.TypeInt,
			Optional:    true,
			Computed:    true,
```

Align verbiage to other Deprecation notices. Also kinda same comment as deprecated resources outside the range of versions we've discussed supporting before (and this one has a replacement field)